crates/polars-utils/src/idx_vec.rs

```diff
@@ -127,6 +142,27 @@ impl Drop for IdxVec {
     }
 }
 
+impl Clone for IdxVec {
+    fn clone(&self) -> Self {
```

Comment:
This still uses the direct allocator instead of `Vec`.

Reply:
Ah, good point.